.changeset/chilled-timers-peel.md

```diff
@@ -0,0 +1,9 @@
+---
+'@finos/legend-application': minor
+'@finos/legend-application-query': minor
+'@finos/legend-art': minor
+'@finos/legend-graph': minor
+'@finos/legend-application-studio': minor
+---
+
+add indicator for system elements ([#457](https://github.com/finos/legend-studio/issues/457)).
```

Comment:
Let's capitalize the message here. Also, this does not fully capture this change since you also show indicator for `generated`, `dependencies`, etc. as well.

Also, this change really just happen in `@finos/legend-art` for `PackageableElementOption`, so we should split the changeset and perhaps mention `PackageableElementOption` in particular.

Reply:
I see! Made two changesets though not sure if I should change description on other one